internal/xds/translator/testdata/in/xds-ir/jsonpatch-with-jsonpath.yaml

```diff
@@ -0,0 +1,163 @@
+envoyPatchPolicies:
```

Comment:
can we add examples with filters, which is the real value prop for using jsonpatch over path since its hard to figure out index

Reply:
Will add this test. It makes clear, that you do not need to go through all parent nested properties and you can select explicitly the route by name, which is much more reliable then selecting via index:
```
  - type: "type.googleapis.com/envoy.config.route.v3.RouteConfiguration"
    name: "first-listener"
    operation:
      op: "replace"
      jsonPath: "..routes[?(@.name=='second-route')].route.upgrade_configs"
      value:
      - upgrade_type: CONNECT
        connect_config:
          {}
```

but there is already this scenario:
`jsonPath: "..endpoints[*].load_balancing_weight"`
which addresses ALL endpoints, which is not possible right now